apps/eaas-api/src/database/migrations/20231101133440_user.ts

```diff
@@ -0,0 +1,19 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable("users", (table) => {
+    table.uuid("id").defaultTo(knex.raw("gen_random_uuid()")).primary();
+    table.timestamps(true, true);
+    table.string("email").unique().notNullable();
+    table.string("password").notNullable();
+    table.boolean("is_admin").notNullable().defaultTo("false");
+    table.string("phone").notNullable();
+    table.string("name").notNullable();
+    table.boolean("is_email_verified").defaultTo("false");
+    table.string("profile_picture_key");
```

Comment:
We can remove profile_picture_key, name and phone for now.

Should `is_admin` be `is_lawyer` or `is_embalmer`? The other option could be to create a user_type enum with `client` and `embalmer` to match the other table that holds the relationships.